deploy/kubernetes/helm/che/templates/_hostHelper.tpl

```diff
@@ -1,7 +1,9 @@
 {{- define "cheHost" }}
-{{- if .Values.global.isHostBased }}
-{{- printf "master.%s" .Values.global.cheDomain }}
+{{- if eq .Values.global.serverStrategy "default-host" }}
```

Comment:
@guydaichs  Looks like after your changes `readme.md` is outdated. I mean the [following line](https://github.com/eclipse/che/blob/09149b296fcade490ea381eaeec132c654b2dd40/deploy/kubernetes/helm/che/readme.md#L29) 
```
If you must use an ip address (e.g. your corporate policy prevents you from using nip.io), you would also have to set isHostBased to false.
```
Could you please review `readme.md` and update information about servers strategies there?

Reply:
@sleshchenko https://github.com/eclipse/che/pull/10152

Reply:
:+1: Thanks